codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -244,6 +244,9 @@ class FluentClientGenerator(
                     }
                     """,
                     *clientScope,
+                    // TODO(enableNewSmithyRuntimeLaunch): Remove the handle config hack
+                    "Storable" to RuntimeType.smithyTypes(runtimeConfig).resolve("config_bag::Storable"),
+                    "StoreReplace" to RuntimeType.smithyTypes(runtimeConfig).resolve("config_bag::StoreReplace"),
```

Comment:
I cannot find occurrences of these in the above rust code snippet in `rustTemplate`, but assume something is still referring to them?

Reply:
Oh, these can be removed.